torch/quantization/quantize.py

```diff
@@ -89,6 +90,10 @@ def add_observer_(module):
         if type(child) == nnq.FloatFunctional or type(child) == nnq.QFunctional:
             if hasattr(child, 'qconfig') and child.qconfig is not None:
                 child.activation_post_process = child.qconfig.activation()
+        elif type(child) == nnqd.Linear or type(child) == nnq.Linear:
```

Comment:
Why do you need to special case nnq.Linear?

Reply:
It's because nnq.Linear has child module _packed_params and existing code won't assign qconfig to it.

Reply:
You can do this instead:
``` python
_INCLUDE_NUMERIC_SUITE_PROPAGATE_LIST = {
    nnq.Linear._packed_params
}
DEFAULT_NUMERIC_SUITE_COMPARE_MODEL_OUTPUT_WHITE_LIST = (
    set(DEFAULT_MODULE_MAPPING.values())
    | set(DEFAULT_QAT_MODULE_MAPPING.values())
    | set(DEFAULT_DYNAMIC_MODULE_MAPPING.values())
    | set(DEFAULT_MODULE_MAPPING.keys())
    | set(DEFAULT_QAT_MODULE_MAPPING.keys())
    | set(DEFAULT_DYNAMIC_MODULE_MAPPING.keys())
    | _INCLUDE_QCONFIG_PROPAGATE_LIST
    |_INCLUDE_NUMERIC_SUITE_PROPAGATE_LIST
) - _EXCLUDE_QCONFIG_PROPAGATE_LIST
```

Also, can you move the list definitions here to _numeric_suite.py so that default_mappings only has the code specific to quantize/prepare/convert?

Reply:
My previous comment is not correct, the issue should be nnq.Linear has child module _packed_params and add_observer() only add observer to leaf node, so adding nnq.Linear._packed_params to white list may not help in this case.